pcp/Platform.c

```diff
@@ -309,7 +309,8 @@ void Platform_init(void) {
    sts = pmLookupName(pcp->totalMetrics, pcp->names, pcp->pmids);
    if (sts < 0) {
       fprintf(stderr, "Error: cannot lookup metric names: %s\n", pmErrStr(sts));
-      exit(1);
+      Platform_done();
```

Comment:
@natoscott Can you take a look here for PCP? Not sure if this call to `Platform_done()` is correct here.